docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.1'
+services:
+  api:
+    container_name: ax_dapp
+    env_file: .env
+    environment:
+      - DISABLE_TESTS=$DISABLE_TESTS
+      - FIREBASE_API_KEY=$FIREBASE_API_KEY
+    restart: always
+    ports:
+      - 5000:5000
```

Comment:
I was looking at this: https://docs.docker.com/engine/reference/commandline/compose/
and the examples show that the port mappings are specified as a string. If I am not mistaken, we should do that here. It should be like this: `- "5000:5000"`

Reply:
wow good point 👍